packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -339,6 +359,19 @@ export default function ProjectNav({
         </MenuItem>
       </Menu>
 
+      {!isMobile &&
+        admin &&
+        current?.status &&
+        Object.values(ProjectStatus).includes(current?.status) && (
```

Comment:
```suggestion
        Object.values(ProjectStatus).includes(current.status) && (
```

Already checked current above.

Reply:
👍